app/views/admin/enterprises/_change_type_form.html.haml

```diff
@@ -27,10 +27,10 @@
               %monthly-pricing-description{ joiner: "newline" }
 
           %p.description
-            = t('.producer_description_text')
+            = t('.sell_description_text')
             %br
             %br
-            = t('.producer_description_text2')
+            = t('.sell_description_text2')
```

Comment:
They were introduced by commit https://github.com/clairezed/openfoodnetwork/commit/88dea0f2b8c60d846d97dd591000b504d2e171b6 , actually the key `producer_description_text2` is not even defined anywhere... I think we can safely remove the unused keys.